helm/dashboards/dashboards/shared/public/dns.json

```diff
@@ -1920,6 +1920,371 @@
+          },
+          "mappings": [],
+          "min": 0,
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 94
+      },
+      "id": 42,
+      "options": {
+        "legend": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "displayMode": "table",
+          "placement": "right",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(rate(coredns_cache_hits_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\", type=\"success\"}[5m])) / sum(rate(coredns_cache_requests_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\"}[5m]))",
+          "instant": false,
+          "legendFormat": "hits ratio: success",
+          "range": true,
+          "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(rate(coredns_cache_hits_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\", type=\"denial\"}[5m])) / sum(rate(coredns_cache_requests_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\"}[5m]))",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "hits ratio: denial",
+          "range": true,
+          "refId": "B"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "1 - (sum(rate(coredns_cache_hits_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\"}[5m])) / sum(rate(coredns_cache_requests_total{cluster_id=\"$cluster\", organization=\"$organization\", app=\"k8s-dns-node-cache\"}[5m])))",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "miss",
+          "range": true,
+          "refId": "D"
+        }
+      ],
+      "title": "NodeLocal hits and miss ratios",
+      "transformations": [],
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
Is this the same across all installations? I remember using `default` or something like that. Except of that, everything should be fine.

Reply:
yes. default is only the name. the `uid` is required here and is this value for all the installations.